### DISCUSSION TITLE
轻涟 ( La vaguelette )　Update

### DISCUSSION THREAD

**Discussion by mirai-mamori** [mirai-mamori/Sakurairo#930] [公告]
![banner-ver2 7](https://s.nmxc.ltd/sakurairo_vision/@2.7/readme/banner-ver2.7.webp)

![cn-ver2 7](https://s.nmxc.ltd/sakurairo_vision/@2.7/readme/cn.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/mirai-mamori/Sakurairo/releases/tag/2.7.0'>轻涟 ( La vaguelette )　Update</a>.</em>

**Comment by muyuanlove** [comment]
突然更新捏~(￣▽￣)~*

**Comment by kemomi** [comment]
芙芙可爱呢
